helm_lib/charts/deckhouse_lib_helm/templates/_csi_controller.tpl

```diff
@@ -175,7 +175,7 @@ spec:
       {{- include "helm_lib_priority_class" (tuple $context "system-cluster-critical") | nindent 6 }}
```

Comment:
replicas: 1 -> replicas: {{ is_ha -> 1 or 2 }}

Reply:
This can affect other CSI controllers, currently they are all in one replica.

Reply:
Added to backlog